_build/templates/default/sass/login.scss

```diff
@@ -217,6 +217,36 @@ input[type="checkbox"] {
   }
 }
 
+#modx-login-btn {
+  border: 1px solid transparent;
+  &:hover {
+    background-color: $white;
+    color: $primary2;
+    box-shadow: none;
+    border: 1px solid $primary2;
+    transition: color .2s ease-out,
+    background-color .2s ease-out,
+    border .2s ease-out;
+  }
+  &:focus{
+    box-shadow: none;
+  }
+}
+
+button#modx-fl-btn {
```

Comment:
Here already selector by ID, no need to add the listing of all buttons before finding a proper element with the specified identifier. And should be combined with the first rule.